woocommerce/woocommerce
# Add UTM group to product URL on the Discover page

### Description
We want to add more data on where items are positioned on the Discover page in the in-app marketplace. To achieve this, we're adding the `utm_group` parameter to the URL of those items to identify the group.


### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes # https://github.com/Automattic/woocommerce.com/issues/18914

### Changes in this PR
- Added `utm_category`  to the Product URL displayed on the Discover page.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Checkout this branch and build.
2. Navigate to the [Extensions page](http://localhost:8888/wp-admin/admin.php?page=wc-admin&path=%2Fextensions). 
3. Confirm the products listed under the discover tab have the `utm_category` parameter in the URL. 
4. Navigate to a different tab, such as `Themes`. 
5. Confirm the products listed under the discover tab don't have the `utm_category` parameter in the URL. 




### Changelog entry




<details>

#### Significance




#### Type




#### Message 
Add UTM group to product URL on the Discover page
#### Comment 

</details>
